src/main/java/nl/tudelft/scrumbledore/Constants.java

```diff
@@ -97,7 +97,11 @@ private Constants() {
   public static final String GAME_EXITBUTTON = "Exit";
 
   // Defining text needed for the display of dialog and handling boxes within the user interface.
+  public static final String GAMEWIN_HEADER = "SCRUMMASTER";
   public static final String GAMEWIN_DIALOG = "You beat the game, congratulations!";
+  public static final String GAMEWIN_POINTS = "You scored ";
+  public static final String GAMEWIN_HIGHSCORE = " points with a session high score of ";
```

Comment:
Maybe we could replace these two constants with one line, containing tokens that should be replaced with the score and highscore?
<organisation